services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy

```diff
@@ -496,24 +499,24 @@ class LinkSpec extends BaseSpecification {
         getIsl().srcSwitch.dpId | -3               | getIsl().dstSwitch.dpId | -4               | "src_port & dst_port"
     }
 
-    def "Isl is able to properly fail when both src and dst switches suddenly disconnect"() {
+    def "Link is able to properly fail when both src and dst switches suddenly disconnect"() {
         requireProfiles("virtual")
 
-        given: "An ISL under test"
+        given: "An active ISL"
         def isl = topology.islsForActiveSwitches.first()
 
-        when: "Source and destination switches of the ISL suddenly disconnect"
+        when: "Source and destination switches of the link suddenly disconnect"
```

Comment:
I doesn't really matter, but I don't fully agree with these renames. If you consider `link` to be a bi-directional entity, then it shouldn't have neither `source` nor `destination`.

Reply:
Fixed.